.gitignore

```diff
@@ -5,3 +5,5 @@ log
 *.pid
 npm-debug.log
 config/index.js
+.vscode
+cordlr.json
```

Comment:
I guess the cordlr.json should never be in this folder but we can keep it in here